RecoTracker/LSTCore/interface/EndcapGeometryBuffer.h

```diff
@@ -0,0 +1,56 @@
+#ifndef RecoTracker_LSTCore_interface_EndcapGeometryBuffers_h
+#define RecoTracker_LSTCore_interface_EndcapGeometryBuffers_h
+
+#include <map>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <vector>
+#include <stdexcept>
+
+#include "RecoTracker/LSTCore/interface/Constants.h"
+
+namespace lst {
+
+  struct EndcapGeometryDev {
```

Comment:
Calling this class `EndcapGeometryView` would be more consistent with the nomenclature we have elsewhere (e.g. in `PortableCollection`).

(same comment for all other classes that follow this pattern)